ipaserver/plugins/vault.py

```diff
@@ -55,7 +55,9 @@
 """) + _("""
 Manage vaults.
 """) + _("""
-Vault is a secure place to store a secret.
+Vault is a secure place to store a secret. One vault can only
+store one secret. Contents of vault would be overwritten if
+at all present.
```

Comment:
I suggest a small tweak to the text for clarify: instead of *Contents of vault would be overwritten if at all present*, 
perhaps *When archiving a secret in a vault, the existing secret (if any) is overwritten.*

Otherwise, ACK.